binding.go

```diff
@@ -458,6 +454,10 @@ func bindForm(req *http.Request, userStruct FieldMapper, formData map[string][]s
 				}
 				continue
 			}
+
+			if len(strs) == 0 {
```

Comment:
Hi @bhcleek, since `if len(strs) ...` is moved to bottom of this block, does this means that those who implements Binder interface need to check the length of the `strVals`?

```go
type MyBinder map[string]string

func (t MyBinder) Bind(fieldName string, strVals []string) error {
	if len(strVals) == 0 {
	    # do something here
	}
	t["formData"] = strVals[0]
	return nil
}
```